src/context-menu-builder.js

```diff
@@ -234,41 +234,42 @@ export default class ContextMenuBuilder {
 
     // Ensure that we have valid corrections for that word
     let corrections = await this.spellCheckHandler.getCorrectionsForMisspelling(menuInfo.misspelledWord);
-    if (!corrections || !corrections.length) {
-      return menu;
```

Comment:
because of this `return`, the Add to Dictionary item on the context menu would only show up if there were other possible corrections. In many cases there are not other corrections, but a user still wants to add a word to the dictionary.